platform/linux-generic/arch/aarch64/odp_crypto_armv8.c

```diff
@@ -93,6 +106,14 @@ struct odp_crypto_global_s {
 
 static odp_crypto_global_t *global;
 
+typedef struct crypto_local_t {
+	uint8_t input[_ARM_CRYPTO_MAX_DATA_LENGTH];
+	uint8_t digest[_ARM_CRYPTO_MAX_DIGEST_LENGTH];
+	uint8_t output[_ARM_CRYPTO_MAX_DATA_LENGTH + _ARM_CRYPTO_MAX_DIGEST_LENGTH];
```

Comment:
What is the + _ARM_CRYPTO_MAX_DIGEST_LENGTH for?

Wouldn't it be more efficient to have just single buffer here and perform in-place operation (which the library claims to support) in those cases where the current code uses both the input & output buffer?

Reply:
I had taken the reference of AArch64cryptolib which wrote its test cases with separate buffers so I will explore the option of using the same buffer for input and output and see what happens.